### DISCUSSION TITLE
What's the current state of GreenThreads experiment?

### DISCUSSION THREAD

**Discussion by En3Tho** [dotnet/runtime#81462] [Q&A] [answered]
Hello! I've been skimming through repo "news" (I mean prs, discussions, issues etc) and I've noticed that there were no updates of GreenThreads branch for 2 months now.

Latest ones are benchmarks and seeing how there was no further work I guessed the results of those benchmarks were not really acceptable?

Obviously this is only a guess. Was the experiment stopped or is it just on pause?

Is there any kind of summary that you can share with community about current state of this feature and what valuable key points (meaning why's, how's, what's, etc) you've got from this experiment currently?

**Comment by am11** [comment]
It's progressing in the runtimelab repo:

Tracking issue: https://github.com/dotnet/runtimelab/issues/2057
Branch: https://github.com/dotnet/runtimelab/tree/feature/green-threads

**Comment by En3Tho** [comment]
I'm not sure why this was selected as an answer @teo-tsirpanis. If it is then clearly not for a question I've asked. I already stated that there were no updates for this branch for 2 months (meaning I know where the branch is located and I've checked it out before writing this).

**Reply by am11** [reply]
There is also a link to tracking issue which I thought what you were looking for. If it wasn't then never mind.

**Reply by En3Tho** [reply]
@am11 Thank you for the provided link but I already knew about it. My question was mainly about why development of this feature was (seemingly) stopped/paused. And it's very peculiar that it stopped exactly after benchmarks code was added to the branch.

**Comment by mangod9** [comment, marked as answer]
Hi @En3Tho,

This was a time boxed experiment late last year to build a very basic prototype to get some functional and perf measurements done.  We plan to publish an update to the runtimelab issue with the learnings soon. Thx!

**Reply by En3Tho** [reply]
Thanks. It's very peculiar topic and can really affect potential .Net languages of the future
